public/components/metrics/top_menu/metrics_export_panel.tsx

```diff
@@ -66,9 +66,21 @@ export const MetricsExportPanel = ({
   const fetchAllvisualizationsById = async () => {
     const tempVisualizationsMetaData = await Promise.all(
       sortedMetricsLayout.map(async (metricLayout) => {
-        return metricLayout.metricType === 'savedCustomMetric'
-          ? await fetchVisualizationById(http, metricLayout.id, setErrorResponse)
-          : createPrometheusMetricById(metricLayout.id);
+        console.log('metricLayout in fetch: ', metricLayout);
```

Comment:
Again, read new MetricExportPanel and adjust accordingly.

Reply:
yes, modified it